lib/trento.pm

```diff
@@ -0,0 +1,371 @@
+# SUSE's openQA tests
+#
+# Copyright 2017-2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Functions for Trento tests
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+## no critic (RequireFilenameMatchesPackage);
+
+=encoding utf8
+
+=head1 NAME
+Trento test lib
+
+=head1 DESCRIPTION 
+Package with common methods and default or constant  values for Trento tests
+
+=head1 COPYRIGHT
+Copyright 2017-2020 SUSE LLC
+SPDX-License-Identifier: FSFAP
+
+=head1 AUTHORS
+QE SAP <qe-sap@suse.de>
+
+=cut
+
+package trento;
+
+use strict;
+use warnings;
+use testapi;
+use mmapi 'get_current_job_id';
+use utils qw(script_retry);
+use File::Basename qw(basename);
+use Carp qw(croak);
+use Exporter 'import';
+
+our @EXPORT = qw(
+  TRENTO_AZ_PREFIX
```

Comment:
You don't need to export constant or ? I think it should be not done.
https://perldoc.perl.org/Exporter#What-Not-to-Export

Reply:
I thought you don't need the constant out of `lib/trento.pm`
It's not explicitly bad, but it should be not done. You can do it if you like I think it's not first case of exported "variable/constant" :man_shrugging:

Reply:
btw define constants on one place, there is another definition of VM_USER at line 123

Reply:
if the constant is not needed in a different file, don't export it.
And if you export something, prefer `@EXPORT_OK` over `@EXPORT` to only import on request.

Reply:
There are some test files like **deploy_trento.pm** that are using many symbols from lib/trento.pm

Does not changing the overall list of exported symbols from EXPORT to EXPORT_OK result in a long and so difficult to maintain list of symbols in the test files?

Does it make sense to have two export lists, one using EXPORT and the other one using EXPORT_OK?